samples/posix/syslog/src/main.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024, Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <syslog.h>
+
+#include <zephyr/sys/util.h>
+
+int main(void)
+{
+	int prios[] = {
+		LOG_EMERG,   LOG_ALERT,  LOG_CRIT, LOG_ERR,
+		LOG_WARNING, LOG_NOTICE, LOG_INFO, LOG_DEBUG,
+	};
+
+	openlog("main", LOG_PID | LOG_CONS | LOG_NOWAIT, LOG_LOCAL7);
+	(void) setlogmask(LOG_MASK(-1));
+
+	ARRAY_FOR_EACH(prios, i) {
```

Comment:
Not sure if it makes sense to use typical for-loop so that this sample is truly portable and can be built on host OS as well

Reply:
Yeah, makes sense.